### ISSUE TITLE
Parquet file format for tabular data

### ISSUE THREAD

**Issue by cpalazzi** [calliope-project/calliope#694]
### What can be improved?

Parquet is a fast and highly compressible open source file type for tabular data. Compared to csvs it can save around 10x the storage space. It is already integrated with Pandas and Xarray so should be a very beneficial feature without too much to change. 

### Version

v0.6.10

**Comment by brynpickering** [comment]
I don't believe that it is integrated into xarray, but certainly useful to consider for loading in with pandas.
